src/main/java/com/example/eventorganizer/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.example.eventorganizer.security;
+
+import com.example.eventorganizer.user.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@AllArgsConstructor
+@EnableWebSecurity
+public class WebSecurityConfig {
+    private final UserService userService;
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception{
+        http.csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(auth ->
+                        auth.requestMatchers("/","/registration").permitAll()
+                                .anyRequest().authenticated())
+                .formLogin(login -> login.loginPage("/login").permitAll());
+        return http.build();
+    }
+    @Bean
+    public DaoAuthenticationProvider daoAuthenticationProvider(){
```

Comment:
You can include UserService and BCryptPasswordEncoder  in this method parameters, as they are only used here